crimsobot/cogs/reactions.py

```diff
@@ -88,7 +88,7 @@ async def fact_add(self, ctx: commands.Context, *, something: CleanMentions) ->
             await ctx.send(embed=error_embed, delete_after=18)
             return
 
-        if not (fact_subject or fact):
```

Comment:
won't fact always have a length of 1 or more? see line 86

Reply:
I hate computers